design/include-cleaner.md

```diff
@@ -0,0 +1,187 @@
+# Include Cleaner
+
+Manually managing includes in a C++ translation unit, especially in the face of
+transitive inclusions, requires a lot of effort. Include Cleaner aims to
+provide diagnostics to keep includes in an
+[IWYU](https://include-what-you-use.org/)-clean state.
+
+Include Cleaner is available in "preview" mode with an incomplete set of
+capabilities and can be enabled through [configuration
+file](/config#UnusedIncludes). If you experience any bugs, please submit a bug
```

Comment:
s/UnusedIncludes/unusedincludes/

(ids are all lower letters no matter what you have in the title :shrug: )